GLTFSDK/Inc/GLTFSDK/Schema.h

```diff
@@ -3,19 +3,102 @@
 
 #pragma once
 
-#include "SchemaJson.h" // autogenerated header
-
 #include <string>
+#include <memory>
 #include <unordered_map>
 
 namespace Microsoft
 {
     namespace glTF
     {
-        namespace GLTFSchema
+        constexpr const char SCHEMA_URI_GLTF[] = "glTF.schema.json";
```

Comment:
I think these would be better nested under another namespace e.g. Schema.

Reply:
Will leave as-is for the time being. Can revisit the need for a Schema namespace when updating the script that generates SchemaJson.h